### PR TITLE
Homework 29

### PR DESCRIPTION
**Pull request by lowkent2me**
Ansible vars:
BRANCH - feature29
VAULT - bank

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -38,6 +40,30 @@
           app_net:
             ipv4_address: 172.16.1.6
 
+      db:
+        image: postgres:14-bullseye
```
[the remark this answers is not in the document]
Comment (lowkent2me) [reply docker-compose.yaml:44]: In fact, docker compose can be deployed locally, but you first need to decrypt the /Ansible/keys/credentials file

[the remark this answers is not in the document]
Comment (ggramal) [reply docker-compose.yaml:44]: i dont have Ansible and i cant run python on my M1 macbook pro so

[the remark this answers is not in the document]
Comment (ggramal) [reply docker-compose.yaml:70]: so anyway i need to decrypt the Ansible password file

[the remark this answers is not in the document]
Comment (lowkent2me) [reply docker-compose.yaml:70]: Not really)
Postgres container uses file as password, and doesn't matter, what it contains.

[the remark this answers is not in the document]
Comment (ggramal) [reply docker-compose.yaml:70]: Ok can you tell me how can i pass this file to psql
Tried
```
$ PGPASSFILE=Ansible/keys/db_password psql -h localhost -p 5433 -U postgres -w
psql: error: connection to server at "localhost" (::1), port 5433 failed: fe_sendauth: no password supplied
```
doesnt seem to work :thinking:

[the remark this answers is not in the document]
Comment (lowkent2me) [reply docker-compose.yaml]: works now
` psql -h localhost -p 5432 -U kot -d bank -w`

[the remark this answers is not in the document]
Comment (ggramal) [reply docker-compose.yaml]:
```

[the remark this answers is not in the document]
Comment (lowkent2me) [reply docker-compose.yaml]: works without using port
connect to db without pass works only in local docker network (all my containers + localhost)
`psql -h localhost -U kot -d bank -w`

[the remark this answers is not in the document]
Comment (ggramal) [reply docker-compose.yaml]: Still not working for me but lets merge this

**Review by ggramal — APPROVED** [review]

**Pull request merged**